components/post_view/post_body_additional_content/post_body_additional_content.tsx

```diff
@@ -111,7 +93,6 @@ export default class PostBodyAdditionalContent extends React.PureComponent {
 
                 return (
                     <YoutubeVideo
-                        postId={this.props.post.id}
```

Comment:
I got error after posting a Youtube link (ex. "https://www.youtube.com/watch?v=a9__D53WsUs"). Might be related to above.
![Screen Shot 2020-11-11 at 12 58 32 AM](https://user-images.githubusercontent.com/5334504/98706225-93c4c400-23b9-11eb-96e4-93010672c033.png)

Reply:
Oh yeah, that's required by the YoutubeVideo's mapStateToProps, but it's not required by the component itself. That'll be resolved properly when that component is migrated to TS, but for now, you can just add it `postId` to the YoutubeVideo's propTypes as required.

Reply:
Thanks for the test and sorry for the bug (and the delay), I did push a fix :+1:

Reply:
Tests are failing I'll look at it

Reply:
That should be ok now